kfs-core/src/main/java/edu/arizona/kfs/gl/businessobject/lookup/GecEntryLookupableHelperServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+package edu.arizona.kfs.gl.businessobject.lookup;
+
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.kuali.kfs.gl.Constant;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.kns.web.struts.form.LookupForm;
+import org.kuali.rice.krad.bo.BusinessObject;
+import org.kuali.rice.krad.service.LookupService;
+
+public class GecEntryLookupableHelperServiceImpl extends org.kuali.kfs.gl.businessobject.lookup.EntryLookupableHelperServiceImpl {
+
+    private LookupService lookupService;
+
+
+    @Override
+    public Collection performLookup(LookupForm lookupForm, Collection resultTable, boolean bounded) {
+
+        @SuppressWarnings("unchecked")//super.getFieldsForLookup() not generified
+        Map fieldsForLookup = new HashMap(lookupForm.getFieldsForLookup());
+        String debitCreditOption = getDebitCreditOption(fieldsForLookup);
+
+        // Reason for overriding:
+        lookupForm.getFieldsForLookup().remove(Constant.DEBIT_CREDIT_OPTION);
+
+        Collection displayList = super.performLookup(lookupForm, resultTable, bounded);
+        updateByDebitCreditOption(resultTable, debitCreditOption);
+
+        return displayList;
+    }
+
+
+    @Override
+    public List<? extends BusinessObject> getSearchResultsUnbounded(Map<String, String> fieldValues) {
+        return this.getSearchResults(fieldValues);
+    }
+
+
+    @SuppressWarnings("unchecked")//BOS retrun type
+    @Override
+    public List<? extends BusinessObject> getSearchResults(Map<String, String> fieldValues) {
```

Comment:
Looking at super.getSearchResults, I couldn't see the difference in the code. Why did we override this method?